lib/beaker/host/windows/exec.rb

```diff
@@ -5,7 +5,7 @@ def reboot
     exec(Beaker::Command.new('shutdown /r /t 0 /d p:4:1 /c "Beaker::Host reboot command issued"'), :expect_connection_failure => true)
   end
 
-  ABS_CMD = 'c:\\\\windows\\\\system32\\\\cmd.exe'
+  ABS_CMD = "/cygdrive/c/WINDOWS/system32/cmd.exe"
```

Comment:
The previous command didn't execute on cygwin windows at all, 
```
$ c:\\WINDOWS\\system32\\cmd.exe /c echo.
-bash: c:\WINDOWS\system32\cmd.exe: command not found
```

I'm unsure if this code path is in any use in our codebase, because it does not seem functional to me.